community/neo4j/src/docs/ops/upgrades.asciidoc

```diff
@@ -1,21 +1,25 @@
 [[deployment-upgrading]]
-Upgrading
-=========
+= Upgrading
+
+:manual-base-url: http://neo4j.com/docs/{neo4j-version}
+:manual-ha-upgrade-guide: {manual-base-url}/ha-upgrade-guide.html
+:manual-cypher-compatibility: {manual-base-url}/cypher-compatibility.html
 
 [IMPORTANT]
 This section describes upgrading a single Neo4j instance.
-Upgrading a Neo4j HA cluster (Neo4j Enterprise) requires a very specific process be followed.
-Please see 
+Upgrading a Neo4j HA cluster (Neo4j Enterprise) requires a very specific procedure be followed.
+Please see
 ifndef::upgradetext[<<ha-upgrade-guide>>.]
-ifdef::upgradetext['Upgrade of a Neo4j HA Cluster' at http://neo4j.com/docs/{neo4j-version}/ha-upgrade-guide.html.]
-
-A database created by an older version of Neo4j will be upgraded during startup when opened by Neo4j {neo4j-version}.
-While this upgrade will occur automatically for minor changes, larger migrations require explicit configuration before Neo4j will start.
+ifdef::upgradetext['Upgrade of a Neo4j HA Cluster' at {manual-ha-upgrade-guide}.]
 
 Each Neo4j version supports upgrading from a limited number of previous versions.
 These upgrades are either automatic, or require explicit configuration to allow them.
+Upgrades between patch releases are automatic.
+Upgrading to the latest patch release is recommended before upgrading to new major or minor versions.
 
-In this release of Neo4j, the following upgrades are permitted.
+A database created by an older version of Neo4j will be upgraded during startup when opened by Neo4j {neo4j-version}.
```

Comment:
At this point the upgrade may be either to {neo4j-version} or to the latest patch release within/under a minor version. How about "... when opened by the target version of Neo4j" or "... the new version of Neo4j"? Then we can use {neo4j-version} in the next sentence.